PL2/PL2-gui/src/main/java/nl/tudelft/pl2016gr2/gui/view/graph/ViewAnnotation.java

```diff
@@ -22,7 +25,13 @@
 
   private static final double MINIMUM_LABEL_WIDTH = 10.0;
   private static final double MINIMUM_LABEL_HEIGHT = 12.0;
+  private static final double MINIMUM_ARROW_WIDTH = 25.0;
```

Comment:
Minimum arrow with is alrgeer than minimum label width? Sure this is desired? (weird overlapping with many small nodes)